masschallenge/django-accelerator
# AC-6521 Display a list of internal program events via the Eventbrite API

#### Changes introduced in [AC-6521](https://masschallenge.atlassian.net/browse/AC-6521)
- add migration to add the "register for events" definition
#### Reproduce the current behavior
- On development, Navigate to [link](http://localhost:8181/admin/mc/navtree/3/change/)
- You should see three nav tree items
#### How to test
- checkout this branch and `make migrate`
- Navigate to [link](http://localhost:8181/admin/mc/navtree/3/change/)
- You should see the new `Register for events` nav tree item
